icecreamshop/icecreamshop/src/main/java/com/summerschool/icecreamshop/repository/ProductRepository.java

```diff
@@ -1,10 +1,10 @@
 package com.summerschool.icecreamshop.repository;
 
 import com.summerschool.icecreamshop.model.Product;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
-public interface ProductRepository extends CrudRepository<Product, Integer> {
+public interface ProductRepository extends JpaRepository<Product, Integer> {
```

Comment:
Please make sure that the ID parameter has the same type as the ID property inside your Product Entity, meaning that you should have Long instead of Integer.